test/spec.sh

```diff
@@ -939,4 +939,9 @@ ble-idioms() {
           $BASH $ZSH $MKSH $BUSYBOX_ASH $OSH_LIST "$@"
 }
 
+ble-features() {
+  sh-spec spec/ble-idioms.test.sh --osh-failures-allowed 0 \
```

Comment:
> Passes now:
>
> http://travis-ci.oilshell.org/jobs/2020-04-18__19-07-12.wwz/_tmp/spec/ble-features.html

I'm really sorry, but I noticed that I made a mistake in moving tests to ble-features. This part should have been

```diff
@@ -943,1,943,1 @@ ble-features() {
-  sh-spec spec/ble-idioms.test.sh --osh-failures-allowed 0 \
+  sh-spec spec/ble-features.test.sh --osh-failures-allowed 0 \
```